grpc-ballerina/tests/resources/generated-sources/tool_test_client_8/helloWorldTimestamp_pb.bal

```diff
@@ -0,0 +1,439 @@
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public client class HelloWorldGreetingCaller {
+    private grpc:Caller caller;
+
+    public isolated function init(grpc:Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendGreeting(Greeting response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextGreeting(ContextGreeting response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public client class HelloWorldTimestampCaller {
+    private grpc:Caller caller;
+
+    public isolated function init(grpc:Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendTimestamp(time:Utc response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextTimestamp(ContextTimestamp response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public type ContextGreetingStream record {|
+    stream<Greeting, error?> content;
+    map<string|string[]> headers;
+|};
+
+public type ContextTimestampStream record {|
+    stream<time:Utc, error?> content;
+    map<string|string[]> headers;
+|};
+
+public type ContextGreeting record {|
+    Greeting content;
+    map<string|string[]> headers;
+|};
+
+public type ContextString record {|
+    string content;
+    map<string|string[]> headers;
+|};
+
+public type ContextTimestamp record {|
+    time:Utc content;
+    map<string|string[]> headers;
+|};
+
+public type Greeting record {|
+    string name = "";
+    time:Utc time;
```

Comment:
How about adding a default value `[0, 0.0d]`?

Reply:
fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/312/commits/356f2ff8695fc42a62a2e06057e774bd180b36a5